audobject/core/object.py

```diff
@@ -56,9 +56,7 @@ def arguments(self) -> Mapping[str, object]:
         signature = inspect.signature(self.__init__)
 
         # non-keyword arguments from __init__
-        names = [
-            p.name for p in signature.parameters.values() if not p.name == "kwargs"
-        ]
+        names = [p.name for p in signature.parameters.values() if p.name != "kwargs"]
```

Comment:
This is an example of simplifying logical expressions using De Morgan's identities